web/public/css/base/_alert.less

```diff
@@ -0,0 +1,61 @@
+.alert {
+  border: none !important;
+  box-shadow: @cd-box-shadow;
+  box-sizing: border-box;
+  .fa {
+    font-size: 1.5em;
+    &:before {
+      position: relative;
+      left: 0.8em;
+      margin-left: -0.5em;
+      top: 0.2em;
+    }
+  }
+  a {
+    color: @cd-white;
+    font-weight: 600;
+    cursor: pointer;
+  }
+  &.alert-right {
+    border-top-right-radius: 0;
+    border-bottom-right-radius: 0;
+    @alert-right-corner-size: 15px;
+    &:after {
+      content: "";
+      height: 0;
+      width: 0;
+      display: block;
+      position: relative;
+      bottom: -(@alert-right-corner-size * 2);
+      margin-left: 100%;
```

Comment:
*twitch*

Either `transformX(100%)`, or `position: absolute; left: 100%;` (and make the parent relative of course)